### ISSUE TITLE
Website doesn't load on Internet Explorer

### ISSUE THREAD

**Issue by alexanmtz** [worknenjoy/gitpay#525]
On our Find a bug and fix program #272, one of our members reported that the site is not opening on Internet Explorer.

### Level
Beginner

**Comment by Shawn-Noruzi** [comment]
Hey @alexanmtz  

I'll work on this, please assign it to me.

**Comment by alexanmtz** [comment]
Yes @Shawn-Noruzi , you're now assigned 👍

**Comment by Shawn-Noruzi** [comment]
Some progress - still has bugs however. 
Things I've tried : 

1.Having babel build with Ie>=11 in mind through modifying the .babelrc file 

2.@babel/preset-env added with babel-core packages 
edited .babelrc to include "defaults" range of support for browsers

Ran into build issue stating conflict with babel-core and @babel-core versions 
Removed babel-core and replaced with @babel-core v 7.0.0 to resolve build issue
Issue remains 

babel -V shows we are working with version 6.26 babel -> Looking through 6.26 docs for Babel to find support for IE.
babel-cli and babel-preset-env are for version 6.x and will not work with babel 7 packages (@babel/xxx)

3.Added proper support for IE11 with Babel 6.26 into .babelc

4.Got a regeneratorRuntime error: 
npm install --save-dev babel-plugin-transform-runtime babel-runtime
Create .babelrc file and add:

{
    "plugins": [
        ["transform-runtime",
        {
            "polyfill": false,
            "regenerator": true
        }]
    ]
}

Ran into Styled-components + promise undefined errors

5. Installed Babel-polyfill to remove promise undefined error 
npm install --save babel-polyfill
import "babel-polyfill"; in index.js at the very TOP before even importing react

> Site now displays but overall UI interaction is slow with some features not even working properly such as the navbar scroll-to-element

Will continue to investigate
